docs/user/resources/06-30-application-connector.md

```diff
@@ -0,0 +1,46 @@
+# ApplicationConnector CustomResourceDefinition (CRD)
+
+The applicationconnectors.operator.kyma-project.io CustomResourceDefinition (CRD) is a comprehensive specification that defines the structure and format used to manage the configuration and status of the Application Connector Module within your Kyma environment.
+
+# Purpose
+
+The CRD serves as the blueprint for creating custom resources (CRs) specific to Application Connectors. These CRs encapsulate the configuration settings and real-time statuses of the Application Connector Module, providing essential insights into the module's behavior on the Kubernetes cluster.
+
+To get the up-to-date CRD and show the output in the `yaml` format, run this command:
+
+```bash
+kubectl get crd applicationconnectors.operator.kyma-project.io -o yaml
+```
+
+## Custom resource specification
+
+| Path | Description | Required |
+| ---- | ----------- | -------- |
+| domainName | The name of the domain that will be used by the virtual service and gateway resources. On gardener cluster this field is autopopulated. | false |
+| appConnValidator.logLevel | Determines the level of verbosity or detail for logging messages generated by __central application connector validator__; one of `debug`, `panic`, `fatal`, `error`, `warn`, `info`, `debug` | false |
```

Comment:
Do we need the word `central` before each component? What's the difference between the central Application Connectivity Validator and the regular one?

Reply:
This is the remains of old architecture where each application (registered system) was handled by its own validator and gateway instances. It was replaced by central components. I agree that we can stop using 'Central' prefix but we must remain consistent across whole our documentation.

Reply:
Ok, so let's remove it :)